SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -393,6 +394,18 @@ TArray<UClass*> GetAllSupportedClasses()
 			}
 		}
 
+		// Check for replicated GameplayAbilities and print a warning if we find one. The UnrealGDK does not currently support this.
+		if (ClassIt->IsChildOf(UGameplayAbility::StaticClass()))
+		{
+			UClass* AbilityClass = *ClassIt;
+			UGameplayAbility* GameplayAbility = Cast<UGameplayAbility>(AbilityClass->GetDefaultObject());
+
+			if (GameplayAbility->GetReplicationPolicy() == EGameplayAbilityReplicationPolicy::ReplicateYes)
```

Comment:
Is it common to change replication policy to `ReplicateYes` at runtime?
If so, we could additionally warn for any ability that has added replicated properties.

Reply:
No shouldn't be. They're instanced from the CDO directly, and used immediately. Modifying the replication state after activation wouldn't function correctly.